list/list.go

```diff
@@ -16,7 +18,7 @@ import (
 // Usage text that will be displayed as command line help text when using the
 // `help list` command
 var Usage = `
-USAGE: %s list [-config <s3config-file>] [prefix] (-url <uri> --datasets) (-url <uri> --dataset <dataset-id>)
+USAGE: %s list [--config <s3config-file>] [prefix] (-url <uri> --datasets) (-url <uri> --dataset <dataset-id>) [--bytes]
```

Comment:
I think for all flags in `sda-cli` (or most of them?), both `--` and `-` work. And in the description of argument, e.g. 

```
Command line arguments:
  -bytes
    	Print file sizes in bytes (not human-readable format).
  -config string
    	S3 config file to use for listing.
  -dataset string
    	List all files in the specified dataset.
  -datasets
    	List all datasets in the user's folder.
  -url string
    	The url of the sda-download server

    [prefix]
        The location/folder of the s3 to list contents.
```
it's always single dash `-` is used. So no need to change to `--config` in this case. Otherwise, we need to make the usage consistent at all places.

Reply:
Good catch, I didn't notice that I added that extra dash. Will fix.